### DISCUSSION TITLE
How to Autodesk Maya offline documentation to Zeal docset, I have tried a lot but all failed?

### DISCUSSION THREAD

**Discussion by chuckbruno** [zealdocs/zeal#1593] [General]
I have tried a lot but still cannot make it successful, only make the command htmls to docset, I don't have any idea now, Could someone help me, really appreciate.

**Comment by trollixx** [comment]
I don't believe there is an existing docset. I could only find https://github.com/sonictk/maya_cpp_reference_docset, which is pretty old. You'd need to generate one yourself. Follow [this guide](https://kapeli.com/docsets), or maybe [doc2dash](https://github.com/hynek/doc2dash) can help.
